requirements.txt

```diff
@@ -8,3 +8,11 @@ tomli==2.2.1; python_version <= '3.10'
 # hipBLASLt deps
 joblib>=1.4.2
 msgpack>=1.1.0
+
+# pytorch test deps
+pytest>=8.0.0
+pytest-xdist>=3.5.0
+numpy>=1.24.0
+psutil>=5.9.0
+expecttest>=0.2.0
+hypothesis>=6.75.0
```

Comment:
Move these to a new `external-builds/pytorch/requirements-tests.txt` or (preferred?) install `pytorch/requirements.txt` (https://github.com/pytorch/pytorch/blob/main/requirements.txt) so we don't need to maintain our own list of requirements (which could easily get out of sync)

Reply:
I can keep it in here: requirements-tests.txt

Reply:
Done!